h/static/scripts/controllers/search-bar-controller.js

```diff
@@ -8,10 +8,20 @@ var AutosuggestDropdownController = require('./autosuggest-dropdown-controller')
 var SearchTextParser = require('../util/search-text-parser');
 var stringUtil = require('../util/string');
 
+
 const FACET_TYPE = 'FACET';
 const TAG_TYPE = 'TAG';
+const GROUP_TYPE = 'GROUP';
 const MAX_SUGGESTIONS = 5;
 
+
+/**
+ * standardized normalization pattern
+ */
+var normaliseStr = function(str){
```

Comment:
Nit: Should use US English spelling for consistency ("normalize"). Would also benefit from a comment about why normalization is done, eg. "Normalize a string for use in comparisons of user input with a suggestion. This causes differences in unicode composition and combining characters/accents to be ignored."